detector/test_helper.go

```diff
@@ -0,0 +1,48 @@
+package detector
+
+import (
+	"reflect"
+
+	"github.com/hashicorp/hcl/hcl/ast"
+	"github.com/hashicorp/hcl/hcl/parser"
+	"github.com/wata727/tflint/config"
+	"github.com/wata727/tflint/evaluator"
+	"github.com/wata727/tflint/issue"
+)
+
+type TestDetector struct {
```

Comment:
__[golint]__
* [golint] exported type TestDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->